stubs/setuptools/setuptools/_distutils/command/check.pyi

```diff
@@ -1,12 +1,11 @@
 from typing import Any
-from typing_extensions import TypeAlias
 
-from ..cmd import Command
+import docutils.utils
 
-_Reporter: TypeAlias = Any  # really docutils.utils.Reporter
+from ..cmd import Command
 
 # Only defined if docutils is installed.
```

Comment:
We can remove this comment

Reply:
The comment is still true. I think it has value if someone wonders why its in the stub but not in their runtime.